src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AdjudicationsResourceTest.java

```diff
@@ -329,4 +327,53 @@ public void returns403IfInvalidRole() {
             assertThatStatus(response, 403);
         }
     }
+
+    @Nested
+    public class GetProvenAdjudications {
```

Comment:
A slight issue here: This file covers int tests for the AdjudicationsResource class. This test ought to be in whatever file covers `BookingResource` integration tests.

Reply:
I'll move it